aws/resource_aws_db_parameter_group.go

```diff
@@ -277,8 +277,25 @@ func resourceAwsDbParameterGroupUpdate(d *schema.ResourceData, meta interface{})
 			}
 		}
 
+		toRemove := map[string]*rds.Parameter{}
+
+		for _, p := range expandParameters(os.List()) {
+			if p.ParameterName != nil {
+				toRemove[*p.ParameterName] = p
+			}
+		}
+
+		for _, p := range expandParameters(ns.List()) {
+			if p.ParameterName != nil {
+				delete(toRemove, *p.ParameterName)
+			}
+		}
```

Comment:
nice 👍 I forgot since technically we're expanding user input here (in the rds cluster resource) and the "name"  field within the `parameter` block is `Required`, we shouldn't ever come across a `nil` (as well i see the `exandParameters` actually skips for cases "name" is an empty string) but i prefer erring on the side of caution when it comes to pointers.